### DISCUSSION TITLE
dynamic illumination in avatar case

### DISCUSSION THREAD

**Discussion by cjlong1213** [google/filament#7413] [General]
**Is your feature request related to a problem? Please describe.**
We use filament in our avatar case, currently the global light and shadow effect looks not good compare to apple Memoji.

ours:
![cbe57a11-d517-4dc7-8142-3129f10dc535](https://github.com/google/filament/assets/15936033/1bf21c5f-df30-41db-9b5b-f625f7c50fa8)

Memoji:
<img width="680" alt="1c902ac9-9584-4207-a1ee-dc963e174df5" src="https://github.com/google/filament/assets/15936033/28d26514-326a-4e49-ada8-fcab0dfa1a9d">

Soft shadow is discussing at [#7397](https://github.com/google/filament/issues/7397), in addition to this, is there any post effect which can improve light and shadow effect? Ambient occlusion?

**OS and backend**
Android OpenGLES

**Comment by romainguy** [comment]
Ambient occlusion would help but you may also want to try a diffent IBL. Please use a Discussion for questions like these.

**Reply by cjlong1213** [reply]
The IBL we currently use as below:
![ibl_1_ibl_rgba](https://github.com/google/filament/assets/15936033/9f1d4ce9-ce84-4dc6-8838-8b95ba27bfc5)
(can not upload ktx file because of company policy)

We tried many HDRI as IBL but the results are not good. It seems that we need an area light in front of avatar which now replace by IBL.
How can we get or generate the high quality of IBL file(ktx), any suggestion?
